### PR TITLE
Add beepboop overlay

### PR DESCRIPTION
**Pull request by IgorDrozniak**

### CODE REVIEW

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -5501,5 +5501,18 @@
     <source type="git">git@github.com:kabili207/zyrenth-overlay.git</source>
     <feed>https://github.com/kabili207/zyrenth-overlay/commits/master.atom</feed>
   </repo>
+  <repo quality="experimental" status="unofficial">
+  <name>beepboop</name>
```
Comment (mgorny) [inline files/overlays/repositories.xml:5505]: Please match the indentation to other entries.

File: files/overlays/repositories.xml [hunk 2]
```diff
@@ -5501,5 +5501,18 @@
...
+  <description lang="en">beepboop - some useless gentoo voerlay</description>
+  <homepage>https://gitlab.com/igordrozniak/beepboop.git</homepage>
```
Comment (mgorny) [inline files/overlays/repositories.xml:5507]: I think the canonical homepage URL is without `.git`.

File: files/overlays/repositories.xml [hunk 3]
```diff
@@ -5501,5 +5501,18 @@
...
+  <owner type="person">
+    <name>Igor Drożniak</name>
+    <email>igordrozniak@disroot.org</email>
```
Comment (mgorny) [inline files/overlays/repositories.xml:5510]: This e-mail does not seem to be registered on Gentoo Bugzilla.

File: files/overlays/repositories.xml [hunk 4]
```diff
@@ -5501,5 +5501,18 @@
...
+  </owner>
+  <source type="git">https://gitlab.com/igordrozniak/beepboop.git</source>
+  <source type="git">git+ssh://git@gitlab.com/igordrozniak/beepboop.git</source>
+  <feed>https://gitlab.com/igordrozniak/beepboop/commits/master.atom</feed>
+</repo>
+
```
Comment (mgorny) [inline files/overlays/repositories.xml:5516]: Please do not add extra whitespace.

**Review by mgorny — CHANGES_REQUESTED** [review]

File: files/overlays/repositories.xml (hunk 4 above)
Comment (IgorDrozniak) [reply files/overlays/repositories.xml]: fixed

File: files/overlays/repositories.xml (hunk 3 above)
Comment (IgorDrozniak) [reply files/overlays/repositories.xml]: fixed

File: files/overlays/repositories.xml (hunk 2 above)
Comment (IgorDrozniak) [reply files/overlays/repositories.xml]: fixed

File: files/overlays/repositories.xml (hunk 1 above)
Comment (IgorDrozniak) [reply files/overlays/repositories.xml]: fixed

File: files/overlays/repositories.xml [hunk 5]
```diff
@@ -5501,5 +5501,17 @@
     <source type="git">git@github.com:kabili207/zyrenth-overlay.git</source>
     <feed>https://github.com/kabili207/zyrenth-overlay/commits/master.atom</feed>
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>beepboop</name>
+    <description lang="en">beepboop - some useless gentoo overlay</description>
+    <homepage>https://gitlab.com/igordrozniak/beepboop</homepage>
```
Comment (mgorny) [inline files/overlays/repositories.xml:5507]: This and sources still seem to differ in case, causing unnecessary redirect.

**Comment by jsmolic** [comment]
ping

**Comment by jsmolic** [comment]
Closing due to inactivity. Feel free to open a new PR.

**Pull request closed without merging**